### PR TITLE
NIAD-379: acg acceptance type2 first - additional test data

### PR DESCRIPTION
**Pull request by bartek-sarul**

### CODE REVIEW

File: src/intTest/resources/outbound_uat_data/acceptance/type2-first-mandatory_1.edifact.dat [hunk 1]
```diff
@@ -0,0 +1,18 @@
+UNB+UNOA:2+TES5+XX11+200610:1438+00000001'
```
Comment (mattd-kainos) [inline src/intTest/resources/outbound_uat_data/acceptance/type2-first-mandatory_1.edifact.dat]: Identical to `type2-first-mandatory.edifact.dat`?

File: src/intTest/resources/outbound_uat_data/acceptance/type2-first-mandatory_1.fhir.json [hunk 2]
```diff
@@ -0,0 +1,61 @@
+{
```
Comment (mattd-kainos) [inline src/intTest/resources/outbound_uat_data/acceptance/type2-first-mandatory_1.fhir.json:1]: Identical to `type2-first-mandatory.fhir.json`?

File: src/intTest/resources/outbound_uat_data/acceptance/type2-first-mandatory_2.edifact.dat [hunk 3]
```diff
@@ -0,0 +1,18 @@
+UNB+UNOA:2+TES6+YY21+200610:1438+00000001'
```
Comment (mattd-kainos) [inline src/intTest/resources/outbound_uat_data/acceptance/type2-first-mandatory_2.edifact.dat]: 👍 to make sure none of these things have been hardcoded

File: src/intTest/resources/outbound_uat_data/acceptance/type2-first-mandatory_1.edifact.dat (hunk 1 above)
Comment (bartek-sarul) [reply src/intTest/resources/outbound_uat_data/acceptance/type2-first-mandatory_1.edifact.dat]: That was a test case added by @grzegorz-robak  but just renamed by me

File: src/intTest/resources/outbound_uat_data/acceptance/type2-first-mandatory_1.fhir.json (hunk 2 above)
Comment (bartek-sarul) [reply src/intTest/resources/outbound_uat_data/acceptance/type2-first-mandatory_1.fhir.json:1]: That was a test case added by @grzegorz-robak but just renamed by me

**Review by mattd-kainos — APPROVED** [review]

**Pull request merged**